src/metabase/models/permissions.clj

```diff
@@ -408,7 +408,7 @@
                              [:= :group_id (u/the-id group-or-id)]
                              (into [:or
                                     [:like path (h2x/concat :object (h2x/literal "%"))]]
-                                   (map (fn [path-form] [:like :object (str path-form "%")])
+                                   (map (fn [path-form] [:or [:like :object (str path-form "%")]])
```

Comment:
It looks like these `:or` expressions have only a single clause? Don't think they're needed.